dist/css/bootstrap.css

```diff
@@ -2320,42 +2468,35 @@ progress {
   --bs-form-file-height: calc(1.5em + 1rem + 2px);
   font-size: 1.25rem;
 }
-
 .form-file-lg .form-file-text,
 .form-file-lg .form-file-button {
   padding: 0.5rem 1rem;
 }
 
 .form-range {
   width: 100%;
-  height: 1.4rem;
+  height: calc(1rem + 0.5em);
```

Comment:
Caused by https://github.com/twbs/bootstrap/commit/1af79bbc5267428af9fcb4a8fc9e6ab0b30328f4. We might need to look into the other effects of the unit change here (not related to this PR btw)

Reply:
Yeah, that's why I decided to split the dist commits so that we can spot the dart-sass only changes.

Reply:
Opened https://github.com/twbs/bootstrap/pull/31746 to fix the unit